app/config/AppConfig.scala

```diff
@@ -27,15 +28,10 @@ import scala.concurrent.duration.FiniteDuration
 @Singleton
 class AppConfig @Inject() (config: Configuration, servicesConfig: ServicesConfig) {
 
-  private lazy val contactHost                  = config.getOptional[String]("contact-frontend.host").getOrElse("")
-  private lazy val contactFormServiceIdentifier = config.get[String]("appName")
-
   lazy val managerEnrolment: String               = config.get[String]("auth.enrolments.manager")
   lazy val teamEnrolment: String                  = config.get[String]("auth.enrolments.team")
   lazy val readOnlyEnrolment: String              = config.get[String]("auth.enrolments.read-only")
   lazy val checkEnrolment: Boolean                = config.get[String]("auth.enrolments.enabled").toBoolean
-  lazy val reportAProblemPartialUrl               = s"$contactHost/contact/problem_reports_ajax?service=$contactFormServiceIdentifier"
-  lazy val reportAProblemNonJSUrl                 = s"$contactHost/contact/problem_reports_nonjs?service=$contactFormServiceIdentifier"
```

Comment:
these can be removed based on the comments I think

Reply:
Yh all those 4 already removed can be removed. Also, the added import `import com.sun.imageio.plugins.common.I18N.getString` can also be removed.